Alpaca.Markets/RestClient.Polygon.cs

```diff
@@ -58,6 +95,7 @@ public sealed partial class RestClient
         /// <param name="limit">Paging - maximal number of historical trades in data response.</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
         /// <returns>Read-only list of historical trade information.</returns>
+        [Obsolete("This version of ListHistoricalTradesAsync will be deprecated in a future release.", true)]
```

Comment:
If you mark something with the `Obsolete` attribute it's a good idea to start with a warning and convert it into error only in the next release.